OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignalRestClient.java

```diff
@@ -52,7 +52,7 @@ void onFailure(int statusCode, String response, Throwable throwable) {}
 
    private static final String OS_API_VERSION = "1";
    private static final String OS_ACCEPT_HEADER = "application/vnd.onesignal.v" + OS_API_VERSION + "+json";
-   private static final String BASE_URL = "https://api.onesignal.com/";
+   private static final String BASE_URL = "http://192.168.1.86:3000/api/v1/";
```

Comment:
undo

Reply:
reverting the local testing commit but leaving in the history so I can add it back easier. I will remove this before merging. For the bug bash I will make this target staging instead